core/src/epicli/data/common/ansible/playbooks/roles/applications/templates/pgbouncer/03-deployment.yml.j2

```diff
@@ -68,6 +68,19 @@ spec:
               subPath: pgpool_postgres_password
             - name: pgpool-shared-init-dir
               mountPath: /epiphany/shared_dir
+      affinity:
```

Comment:
While this will work for the deployment I think it will still create some resources like namespaces etc. For applications in general we need some mechanism that will avoid installation of anything else then the auth-service on ARM:

![image](https://user-images.githubusercontent.com/11056665/114511624-6dd96f00-9c38-11eb-9579-10cb85ffb923.png)